app/assets/javascripts/backbone/views/timeline_table_view.coffee

```diff
@@ -116,3 +118,24 @@ class MS.Views.TimelineTableView extends Backbone.View
 
   duplicateStep: (e) ->
     @duplicateStepModal.show(e, @stepIdFromEvent(e), @refresh.bind(@))
+
+  showRelatedSteps: (e) ->
+    $step = @$(e.currentTarget).closest('tr').find('.step-start-date')
+    currentId = $step.data('id')
+    precedentId = $step.data('precedent-id')
+    $table = $step.closest('tbody')
+
+    # Highight start and end date of active step
+    $step.addClass('highlighted')
+    $step.closest('tr').find('.step-end-date').addClass('highlighted')
```

Comment:
Now that I see this I think it worked slightly different in the old system:

* If you mouseover a start date, it highlights the precedent step's end date (if exists)
* If you mouseover an end date, it hightlights the subsequent steps' start date (if they exist) 
* If you mouseover any other cell, nothing happens

Can we adjust it this way please? Sorry for not being clearer. I think you have all the pieces here, just a matter of rearranging them.